<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-424KU7A" failures="1" tests="1" name="Test" time="83.978" errors="0" timestamp="2024-01-07T16:35:47 IST">
  <testcase classname="testCases.TC_003_LoginDDTTest" name="Verify_LoginDDT" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.io.FileNotFoundException: \testData\Opencart_LoginData.xlsx (The system cannot find the path specified)">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: \testData\Opencart_LoginData.xlsx (The system cannot find the path specified)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.io.FileNotFoundException: \testData\Opencart_LoginData.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at utilities.ExcelFileUtility.getRowCount(ExcelFileUtility.java:32)
at utilities.DataProviders.getData(DataProviders.java:15)
... 25 more
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Verify_LoginDDT -->
</testsuite> <!-- Test -->
